charts/orchestrator/values.yaml

```diff
@@ -34,23 +34,55 @@ postgres:
   dataDir: /var/lib/postgresql/data/dbfiles  # where the database data files are stored inside the container
 
 # override Janus image with one with the orchestrator plugin
-backstage: 
+backstage:
+  global:
+    dynamic:
+      includes:
+        - dynamic-plugins.default.yaml
+      plugins:
+        - disabled: false
+          integrity: >-
+            sha512-YbCxiO+3xU+08hky9VJZnu0JrjX8KUNZKxAaRRC0l+WWMl92kN1PbSW7m/msfQ/TR2/MN3UMl5Pgqr/NYNlqIQ==
+          package: '@caponetto-tests/backstage-plugin-orchestrator-backend-dynamic@0.0.2'
```

Comment:
how will we update versions in the future?

Reply:
we can bump the version included here (in the future we'll have to use plugins released from janus-idp org), then once confirmed the compatibility of the plugin with the operator version, the version of the helm chart will have to be bumped as well to allow upgrades of existing installations.

Reply:
do you think it makes sense to make it configurable?

Reply:
The values themselves can be overridden from the command line when installing the helm chart by providing the `--set backstage.global.dynamic.plugins[0].package` - but I'd postpone this consult janus-idp team about they recommendation for plugins upgrade.